fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergMetadata.java

```diff
@@ -389,11 +389,18 @@ public List<PartitionInfo> getPartitions(Table table, List<String> partitionName
                         // Get the partition data/spec id/last updated time according to the table schema
                         StructProjection partitionData = row.get(0, StructProjection.class);
                         int specId = row.get(1, Integer.class);
-                        long lastUpdated = row.get(9, Long.class);
                         PartitionSpec spec = icebergTable.getNativeTable().specs().get(specId);
-                        Partition partition = new Partition(lastUpdated);
                         String partitionName =
                                 PartitionUtil.convertIcebergPartitionToPartitionName(spec, partitionData);
+
+                        long lastUpdated = -1;
+                        try {
+                            lastUpdated = row.get(9, Long.class);
+                        } catch (NullPointerException e) {
```

Comment:
why not check the null in result

Reply:
Iceberg sdk throws the npe.  and I can't check null based on the context.